File: class15/java/AP1501-MenuOperacoes.java
import java.util.Scanner;

public class MenuOperacoes {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("1 - Somar");
        System.out.println("2 - Subtrair");
        System.out.println("3 - Multiplicar");
        System.out.print("Escolha uma opcao: ");
        int opcao = sc.nextInt();
        System.out.print("Primeiro numero: ");
        double a = sc.nextDouble();
        System.out.print("Segundo numero: ");
        double b = sc.nextDouble();
        switch (opcao) {
            case 1: System.out.println("Resultado: " + (a + b)); break;
            case 2: System.out.println("Resultado: " + (a - b)); break;
            case 3: System.out.println("Resultado: " + (a * b)); break;
            default: System.out.println("Opcao invalida");
        }
    }
}

File: class15/cpp/AP1501-MenuOperacoes.cpp
#include <iostream>
int main() {
    int opcao;
    double a, b;
    std::cout << "1 - Somar\n2 - Subtrair\n3 - Multiplicar\n";
    std::cout << "Escolha uma opcao: ";
    std::cin >> opcao;
    std::cout << "Primeiro numero: ";
    std::cin >> a;
    std::cout << "Segundo numero: ";
    std::cin >> b;
    switch (opcao) {
        case 1: std::cout << "Resultado: " << (a + b) << std::endl; break;
        case 2: std::cout << "Resultado: " << (a - b) << std::endl; break;
        case 3: std::cout << "Resultado: " << (a * b) << std::endl; break;
        default: std::cout << "Opcao invalida" << std::endl;
    }
    return 0;
}
